pkg/kubectl/cmd/certificates/certificates.go

```diff
@@ -103,7 +103,7 @@ func (o *CertificateOptions) Validate() error {
 }
 
 func NewCmdCertificateApprove(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
-	options := CertificateOptions{
+	o := CertificateOptions{
```

Comment:
There should be a constructor rather than directly creating this, similar to how we do it in other commands.